### PR TITLE
Adding Go quickstart example for Drive Activity v2 API.

### PR DESCRIPTION
**Pull request by jsmeredith**
Style is intentionally similar to the other Drive Activity v2 API quickstarts (i.e. same helper methods, etc.).

### CODE REVIEW

**Comment by erickoledadevrel** [comment]
@asrivas Can you take this one? You know Go a lot better than I do.

File: drive/activity-v2/quickstart/quickstart.go [hunk 1]
```diff
@@ -0,0 +1,230 @@
...
+	if actor.User != nil {
+		return getUserInfo(actor.User)
+	}
+	return getOneOf(*actor)
+}
+
+// Returns information for a list of actors.
+func getActorsInfo(actors []*driveactivity.Actor) []string {
+	actorsInfo := make([]string, len(actors))
+	for i := 0; i < len(actors); i++ {
```
Comment (asrivas) [inline drive/activity-v2/quickstart/quickstart.go]: can you change this to a for with range (the for-each)?

File: drive/activity-v2/quickstart/quickstart.go [hunk 2]
```diff
@@ -0,0 +1,230 @@
...
+		}
+		return "fileComment:unknown"
+	}
+	return getOneOf(*target)
+}
+
+// Returns information for a list of targets.
+func getTargetsInfo(targets []*driveactivity.Target) []string {
+	targetsInfo := make([]string, len(targets))
+	for i := 0; i < len(targets); i++ {
```
Comment (asrivas) [inline drive/activity-v2/quickstart/quickstart.go]: same about using a for-each loop

File: drive/activity-v2/quickstart/quickstart.go (hunk 1 above)
Comment (jsmeredith) [reply drive/activity-v2/quickstart/quickstart.go]: Done!

File: drive/activity-v2/quickstart/quickstart.go (hunk 2 above)
Comment (jsmeredith) [reply drive/activity-v2/quickstart/quickstart.go]: Done!

**Pull request merged**